frontend/apps/home/src/app/grid/components/cell-types/grid-cell-background.scss

```diff
@@ -0,0 +1,24 @@
+@import "../../../../assets/styles/palette";
```

Comment:
imo zrobmy jakis jeden scss ktory bedziemy importowac w kazdym scssie komponentu zamiast sciany importow w poszczegolnych plikach ;)

Reply:
na takie rzeczy jak zmienne to zrobimy sobie importy po ~ ;-)

Reply:
a importy z tyldą nie są tylko do css?